canvas_modules/common-canvas/src/object-model/layout-dimensions.js

```diff
@@ -488,7 +488,7 @@ const portsVerticalDefaultLayout = {
 		labelAllowReturnKey: false, // true allows line feed to be inserted into label, "save" to make the return key save the label.
 
 		// An array of decorations to be applied to the node. For details see:
-		// https://github.com/elyra-ai/canvas/wiki/2.4.2-Decoration-Specification
+		// https://github.com/elyra-ai/canvas/tree/main/guide/2.4.2-Decoration-Specification.md
```

Comment:
Same as above.  Have a look at all the links.

Reply:
Hi Matt, I have reverted the docs links to point to existing elyra canvas docs once the new one is deployed I will change these urls pointing to new docs.